crates/types/src/message.rs

```diff
@@ -396,8 +445,12 @@ impl<
             },
             Right(committee_message) => match committee_message {
                 CommitteeConsensusMessage::DAProposal(_) => MessagePurpose::Proposal,
-                CommitteeConsensusMessage::DAVote(_) => MessagePurpose::Vote,
+                CommitteeConsensusMessage::DAVote(_) | CommitteeConsensusMessage::VidVote(_) => {
```

Comment:
Due to some tech debt in the web server design, the VidVote will need to have it's own message purpose so the web server can differentiate it from a QuorumVote.

Reply:
new issue https://github.com/EspressoSystems/HotShot/issues/1702